docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
I propose to leave the installation instructions for PyQt4 because (as you said) there's no harm in having them, but *don't* add further instructions for other Linux distributions and Qt bindings.